src/components/Message.tsx

```diff
@@ -52,7 +52,7 @@ const Message: FC<Props> = ({ message, currentMember, member }) => {
       {!isOwnMessage && (
         <Typography variant="subtitle2">{`${creatorName}`}</Typography>
       )}
-      <Typography variant="body1">{message.body}</Typography>
+      {message.body.split('\n').map((line) => <Typography variant="body1">{line}<br /></Typography>)}
```

Comment:
What about this? https://stackoverflow.com/questions/39325414/line-break-in-html-with-n